Show snowfall accumulation

rebfrank opened the pull request:
I think it would be nicer to show this with the "daily" data; right now it's with current data about
precip probability, wind, and humidity. I basically just swapped it with visibility.

rebfrank committed:
Show snowfall accumulation

I think it would be nicer to show this with the "daily" data; right now it's with current data about
precip probability, wind, and humidity. But, it seems to be working.

rebfrank committed:
Handle undefined precip accumulation

pifantastic commented on public/javascripts/components/weather.js:
```diff
@@ -50,12 +50,12 @@ class Weather extends Component {
               <tr>
-                <th>wind</th>
-                <td>{windSpeed} mph {degreesToDirection(windBearing + 180)}</td>
+                <th>snowfall</th>
```
Can we use `precipType` here?

rebfrank replied:
Pasting screenshot here cuz hangouts isn't uploading...here is an option putting it in the center...though it sounds like maybe you are thinking it's ok to leave it on the right?

![image](https://user-images.githubusercontent.com/29530398/54254709-6eafec00-451a-11e9-87cd-4118688eb904.png)

rebfrank replied:
Sorry, that was the wrong spot to paste this...wasn't paying attention.

pifantastic replied:
We could repurpose the allergies section (which no longer works), e.g.:

<img width="802" alt="Screen Shot 2019-03-12 at 10 09 01 PM" src="https://user-images.githubusercontent.com/126461/54255041-8768d380-4513-11e9-957c-d445a71e6d22.png">

rebfrank commented:
Playing with the summary:

![image](https://user-images.githubusercontent.com/29530398/54255047-f518fd80-451b-11e9-92e3-08cab698e7ee.png)

rebfrank committed:
Show today's human-readable text summary

Show today's summary instead of the precip accumulation, as the summary text seems to include
notable snowfall as well as other types of precipitation. The precipAccumulation field only applies
to snowfall (not other types of precipitation).
